AutoCollection/Statsbeat.ts

```diff
@@ -148,15 +164,52 @@ class Statsbeat {
         this._retryCount++;
     }
 
-    public async trackStatsbeatMetrics() {
+    public async trackShortIntervalStatsbeats() {
+        this._getResourceProvider(async () => {
+            let networkProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion,
+                "attach": this._attach,
+                "instrumentation": this._instrumentations,
+            }
+            this._trackRequestDuration(networkProperties);
+            this._trackRequestsCount(networkProperties);
+            let attachProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "rpid": this._resourceIdentifier,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion
+            }
+            this._statbeatMetrics.push({ name: Constants.StatsbeatCounter.ATTACH, value: 1, properties: attachProperties });
+            await this._sendStatsbeats();
+        });
+    }
+
+    public async trackLongIntervalStatsbeats() {
         this._getResourceProvider(async () => {
-            this._trackRequestDuration();
-            this._trackRequestsCount();
+            let featureProperties = {
+                "os": this._os,
+                "rp": this._resourceProvider,
+                "cikey": this._cikey,
+                "runtimeVersion": this._runtimeVersion,
+                "language": this._language,
+                "version": this._sdkVersion,
+                "attach": this._attach,
+                "feature": this._features,
```

Comment:
The "aad handling" is listed as a feature. Will stats be released as the next beta version?

Reply:
@hectorhdzg is aad handling the same as tracking aad enablement? i check your repo.. look like this is not added anywhere to the feature list. it only happens in tests.

Reply:
We only have AAD feature in beta branch, I'm planning to update beta with latest changes in main for next release including Statsbeat implementation

Reply:
Statbeats will be part of both releases, beta and regular one